### DISCUSSION TITLE
How to get started properly?

### DISCUSSION THREAD

**Discussion by rduzenbury** [GerryDazoo/Slinger#639] [Q&A]
Hi,

I have to say that this looks like a bit of brilliance.  Kudos to you for getting it off the ground.  I've got a slinbox solo that's been collecting dust apparently since 11/7/2009 and I'd like to try to get it working again.  I was able to gather the admin password from the secure.sling.com site.  I was about to plug it in when I started reading about the devices getting bricked.  What's the current guidance there?  

I haven't been able to locate the original player software, and I presume that since sling is out of business, I can't download it from their site, and I don't really care to in any case since I mainly use linux.  I see in the readme that a number of media players have been tested - any favorites?  

Can a "google TV device" stream the slingbox server so I can watch it on the "big TV"?  If so, what are the general steps?

Thank you.

Regards,
Rich

**Comment by bradut** [comment]
Hi Rich,
I would recommend you following the steps in this guide:
https://www.mediafire.com/file/ppw6ep09t99xhu0/Slingbox_ProHD_PC_Unbricking-Setup-Network-Guide.pdf/file
I guess they provide a link to slingplayer 1.5.xxx, if not, you can search for it on this website and will find links to download it.

Also, you may have a look in this thread (and any similar ones):
https://github.com/GerryDazoo/Slinger/discussions/357

**Reply by rduzenbury** [reply]
Thank you!  I'm interested in sling player alternatives since I don't really run windows on my personal network, though it may be useful to try it briefly as a testing tool.

**Reply by GerryDazoo** [reply]
When you say sling player alternatives are you just looking for the media streamer functionality? There's a number of apps that will display the stream from Slinger. What are you running on your network? Linux?

**Reply by rduzenbury** [reply]
Thanks for getting back with me.

I have chromebooks, android phones, ubuntu workstations, as well as a google TV.  I was spinning through the google TV apps over the weekend and I see that VLC is an option - is it a good one, or more appropriately - what non-sling client apps work well?

**Reply by GerryDazoo** [reply]
VLC will work. But any app that will play an H.248 video stream should be OK.

**Reply by rduzenbury** [reply]
Progress report:

I set up my firewall to block all traffic to/from the slingbox, except for dns lookups and dhcp requests.  I plugged in the slingbox and it wasn't looking good.  It looked like the hardware was possibly bad since it wouldn't send a dhcp request and the ethernet port on the switch kept going up and down every few seconds.  I walked away for maybe an hour and when I came back the port was lit and the slingbox looks like its no longer in a reboot loop.  I double checked the firewall and the slingbox does have an IP assigned.

I do see two attempts to access the slingbox kill port, which were blocked by my firewall:

![2023-12-09 16_33_37-Live View _ Log Files _ Firewall](https://github.com/GerryDazoo/Slinger/assets/130404133/efc9f8ae-4ab0-4032-b563-221ed536d983)

Next step will be to try the slinger software but I'm out of time for today.

Thanks again.

**Comment by GerryDazoo** [comment]
A few heads up. Unlike the HD Pros, the Solos CANNOT be un-bricked so it is very important to not  turn on/connect the Solo to your network if there is Internet access until you block TCP port 5678 OUTGOING ( from the Solo, to the Internet). You may need to disconnect your ISP's uplink while you configure your router to do the blocking.

**Reply by rduzenbury** [reply]
Thanks for getting back with me.  Ah, so they are still trying to brick devices.  I'll take appropriate measures.
